core/network/fabric-interop-cc/.env.template

```diff
@@ -0,0 +1,10 @@
+CCID=<ccid-after-installing-chaincode-to-peer>
+CHANNEL=<channel-name>
+PEER=<peer-name>
+ORG=<org-name>
+NETWORK=<network-name>
+DOCKER_IMAGE_NAME=ghcr.io/hyperledger-labs/weaver-fabric-cc
+DOCKER_TAG=1.4.0
+EXTERNAL_NETWORK=<docker-bridge-network>
+COMPOSE_PROJECT_NAME=<docker-compose-project-name>
+COMPOSE_PROJECT_NETWORK=<docker-compose-project-network>
```

Comment:
Can you add some instructions in the README about the purpose of this file and how to populate it?

Reply:
I realised the README was quite outdated, and we need to update it totally. I'll update Readme completely in a separate PR.